core/core.go

```diff
@@ -250,6 +257,32 @@ func initPrivateKey(path string) (p2pcrypto.PrivKey, error) {
 	return nil, err
 }
 
+func initNetworkId(networkID int, meshDB *meshdb.MeshDB) error {
```

Comment:
Let's follow our naming conventions in other parts of the code (and in `libp2p` and _most_ of the standard library) and call this function `initNetworkID` with a capital `D`.

(As an aside, my personal opinion is that the version you have here is correct since "id" is an abbreviation and not an acronym. But nonetheless the convention of using `ID` in Go is pretty well established at this point and some linters will even complain about it.)

Reply:
Will do. I personally agree with you, readability is lost IMO in a case like `NetworkIDDetected`, but of course we should stick to the established convention.